frb_example/pure_dart/dart/lib/bridge_definitions.dart

```diff
@@ -752,6 +853,46 @@ class SumWith {
       );
 }
 
+class TestId {
+  final I32Array2 field0;
+
+  TestId({
+    required this.field0,
+  });
+}
+
+class U8Array1600 extends NonGrowableListView<int> {
+  U8Array1600(Uint8List inner)
+      : assert(inner.length == 1600),
+        super(inner);
+  U8Array1600.unchecked(Uint8List inner) : super(inner);
+  U8Array1600.init() : super(Uint8List(1600));
+}
```

Comment:
uber-nit: what about extracting 1600 to a const

```suggestion
class U8Array1600 extends NonGrowableListView<int> {
  static const arraySize = 1600;
  U8Array1600(Uint8List inner)
      : assert(inner.length == arraySize),
        super(inner);
  U8Array1600.unchecked(Uint8List inner) : super(inner);
  U8Array1600.init() : super(Uint8List(arraySize));
}
```